github/projects.go

```diff
@@ -74,6 +77,15 @@ type ProjectOptions struct {
 
 	// State of the project. Either "open" or "closed". (Optional.)
 	State string `json:"state,omitempty"`
```

Comment:
I believe all three of these optional parameters need to be pointers.
I think I made a mistake in the review of #746 and that we should have used `State *string` here, not `State string`. Can you please change that?

Also, please make the next two parameters pointers as well.